### ISSUE TITLE
German translation: switch from formal "Sie" to informal "Du"

### ISSUE THREAD

**Issue by nikwen** [squarecat/ellie-website#3]
See this for an explanation of the difference: https://www.german-way.com/history-and-culture/german-language/du-and-sie/

The usage of "Sie" feels out of place in the context of how the rest of the copy is written.

**Comment by Jivings** [comment]
Can I just find/replace this?

**Comment by nikwen** [comment]
No, because it comes in a bunch of different grammatical cases and because it changes sentences from 3rd person to 2nd person.
